test/func.js

```diff
@@ -401,16 +410,40 @@ describe('func', function () {
 
               if (itemIndex >= 50) break;
 
-              var item_from_mongo = items[itemIndex];
+              var item_from_elastic = items[itemIndex];
+
               var item_from_array = randomItems[itemIndex];
 
-              if (item_from_mongo.data.item_sort_id != item_from_array.item_sort_id) return done(new Error('descending sort failed'));
+              if (item_from_elastic.data.item_sort_id != item_from_array.item_sort_id) return done(new Error('ascending sort failed'));
             }
 
-            done();
+            //ascending
+            randomItems.sort(function (a, b) {
+
+              return b.item_sort_id - a.item_sort_id;
+            });
+
+            serviceInstance.find(base_path + '/*', {
```

Comment:
Why does this test have /* but the ascending test only had *

Reply:
fixed